### ISSUE TITLE
README Links

### ISSUE THREAD

**Issue by kaufmann42** [kaufmann42/issueist#16]
Update links in README to point to correct urls.

**Comment by kaufmann42** [comment]
blocked by #17

**Comment by b30wulffz** [comment]
Can you assign it to me?

**Comment by kaufmann42** [comment]
@b30wulffz This issue is currently being blocked by publishing. When we get some active links you can open up a new PR with them. Links on the website should also be updated with this issue.
